iati-activities-schema.xsd

```diff
@@ -2131,9 +2128,14 @@
               <xsd:annotation>
                 <xsd:documentation xml:lang="en">
                   A boolean describing the behaviour of the indicator.
-                  It is "true" if the indicator improves from small to
-                  large (e.g. clinics built); false if it improves
-                  from large to small (e.g. cases of a disease).
+                  It is "true" if the indicator improves from small to large (e.g. clinics built); false if it improves from large to small (e.g. cases of a disease).
+                </xsd:documentation>
+              </xsd:annotation>
+            </xsd:attribute>
+            <xsd:attribute name="aggregation-status" type="xsd:boolean" use="optional">
+              <xsd:annotation>
+                <xsd:documentation xml:lang="en">
+                  Boolean flag indicating whether the data in the result set are suitable for aggregation.
```

Comment:
Should `the data [...] are` be `the data [...] is`?

(here and elsewhere)

@IATI/bas